linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/smartcard_configure_ca/ansible/shared.yml

```diff
@@ -1,22 +1,46 @@
-# platform = multi_platform_sle
+# platform = multi_platform_sle,multi_platform_ubuntu
 # reboot = false
 # strategy = restrict
 # complexity = low
 # disruption = low
 
+{{%- if 'sle' in product %}}
+  {{%- set pam_package = "pam_pkcs11" %}}
+{{%- else %}}
+  {{%- set pam_package = "libpam-pkcs11" %}}
+{{% endif %}}
+
 - name: Package facts
   package_facts:
 
+- name: Check if cert_policy entry exists in /etc/pam_pkcs11/pam_pkcs11.conf
+  shell: |
+    set -o pipefail
+    grep "cert_policy" /etc/pam_pkcs11/pam_pkcs11.conf
+    wc -l
+  register: cert_policy_count
+  changed_when: false
+  when: '"{{{ pam_package }}}" in ansible_facts.packages'
+
+- name: Add cert_policy entry if none exist in /etc/pam_pkcs11/pam_pkcs11.conf
```

Comment:
Please, ensure the [Project Style Guides](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#ansible)